net.sourceforge.uhexen2.uhexen2.desktop

```
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Name=Hexen II: Hammer of Thyrion
+Comment=A modern Linux port of Hexen II.
+Exec=glhexen2
+Icon=net.sourceforge.uhexen2.uhexen2.png
```

```suggestion
Icon=net.sourceforge.uhexen2.uhexen2
```

My comment was ignored. It is the **icon name** not the icon *file name*

This is likely why the build failed.

---

Ack, I'd made that change in the `.desktop` file I submitted upstream but not in the one here. Sorry about that.